src/fromager/settings.py

```diff
@@ -17,6 +17,22 @@ def pre_built(self, variant: str) -> set[str]:
         names = p.get(variant) or []
         return set(overrides.pkgname_to_override_module(n) for n in names)
 
+    def url_to_download_sdist_from(self, pkg: str) -> list[str] | None:
+        p = self._data.get("download_source") or {}
+        download_source = p.get(pkg) or {}
+        return download_source.get("url")
+
+    def rename_sdist_to(self, pkg: str) -> str | None:
+        p = self._data.get("download_source") or {}
+        download_source = p.get(pkg) or {}
+        return download_source.get("rename_to")
+
+    def map_version_to(self, pkg: str, version: str) -> str:
```

Comment:
Same here, it's not clear what the version is being mapped to, or why. Maybe `version_override()`?